packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -118,7 +118,8 @@ module.exports = {
           // Point ESLint to our predefined config.
           options: {
             configFile: path.join(__dirname, '../.eslintrc'),
-            useEslintrc: false
+            useEslintrc: false,
+            cache: true
```

Comment:
Thanks for the comments! I realized that i'm searching the cache issues in eslint repo rather than eslint-loader repo. I've sent a pr on tmp dir fallback on eslint-loader repo on https://github.com/MoOx/eslint-loader/pull/154. And yup, I think we should wait until https://github.com/MoOx/eslint-loader/pull/151 merged first.

> We probably don't want this specific setting to be inside of // @remove-on-eject block.

So, should we move the `// @remove-on-eject block.` to the inside of the options?